source/gui/__init__.py

```diff
@@ -53,6 +53,8 @@ def __init__(self):
 		style = wx.DEFAULT_FRAME_STYLE ^ wx.MAXIMIZE_BOX ^ wx.MINIMIZE_BOX | wx.FRAME_NO_TASKBAR
 		super(MainFrame, self).__init__(None, wx.ID_ANY, versionInfo.name, size=(1,1), style=style)
 		self.Bind(wx.EVT_CLOSE, self.onExitCommand)
+
+		# Ensure that this icon is destroyed when this is destroyed as wx doesn't track it (#12243)

```

Comment:
I think this should just duplicated the comment in the Destroy method, it's helpful to know why it won't be automatically destroyed.